InvenTree/templates/js/translated/table_filters.js

```diff
@@ -18,6 +18,30 @@
 */
 
 
+// Construct a dynamic API filter for the "issued by" field
+function constructIssuedByFilter() {
+    return {
+        title: '{% trans "Issued By" %}',
+        options: function() {
+            let users = {};
+
+            inventreeGet('{% url "api-user-list" %}', {}, {
+                async: false,
+                success: function(response) {
+                    for (let user of response) {
+                        users[user.pk] = {
+                            key: user.pk,
+                            value: user.username
+                        };
```

Comment:
Just if you want to learn something more modern, you could do it with reduce like:
```js
response.reduce((users, user) => ({...users, [user.pk]: { key: user.pk, value: user.username }}), {})
```
Or using the map function and Object.fromEntries in combination.

Reply:
Thanks @wolflu05 for the tip. This will all soon be replaced with React - and I am certainly focusing on writing more up-to-date code over there :)